storybookjs/storybook
# fix(react-native): prevent the Navigation bar to stay underneath the keyboard

Issue: N/A

## What I did

Wrapping react-native `<OnDeviceUI>` with a [`<KeyboardAvoidingView>`](https://facebook.github.io/react-native/docs/keyboardavoidingview) prevent the bottom navigation bar to stay underneath the keyboard when a TextInput field is focused. It's an issue only on iOS.

Solution tested on Android and iOS.

before:
![before](https://user-images.githubusercontent.com/20709038/47960226-01210b80-dfce-11e8-8463-1ecf81446b36.png)

after:
![after](https://user-images.githubusercontent.com/20709038/47960225-01210b80-dfce-11e8-9db5-822bde9edfea.png)

## How to test

Is this testable with Jest or Chromatic screenshots? No
Does this need a new example in the kitchen sink apps? `<TextField>` component created to demo this fix (45d81aa6d1d96e0f3cbe8fb81c06aef74e4f8f80. In case it's not needed to have this new component and story, just let me know that I can remove it without any problem)
Does this need an update to the documentation? No

If your answer is yes to any of these, please make sure to include it in your PR.

For maintainers only: Please tag your pull request with at least one of the following:
`["cleanup", "BREAKING CHANGE", "feature request", "bug", "documentation", "maintenance", "dependencies", "other"]`
